### ISSUE TITLE
Phaser template does not work right out-of-the-box

### ISSUE THREAD

**Issue by Nodragem** [DragonBones/DragonBonesJS#56]
Hello,

If you downloaded the demos for phaser.io, chances are that you started a server in `Phaser/Demos/` and went to visit `localhost:xxxx` in your browser... then you just got a black screen.

To resolve the issue, you need to:
1. create a folder `libs` in  `Phaser/Demos`
2. copy-paste the content of `Phaser/2.x/out` in the folder `Phaser/Demos/libs/`
3. change the line 9 of `Phaser/Demos/index.html` from:
```
    <script src="./libs/dragonBones/dragonBones.js"></script>
```
to
```
    <script src="./libs/dragonBones.js"></script>
```

Now refresh the page on your browser and you should have a big robot idling on the screen.

Maybe a pull request could be made based on the above?

**Comment by akdcl** [comment]
Hi, you can read the README.md of the phaser demos.
https://github.com/DragonBones/DragonBonesJS/blob/master/Phaser/Demos/README.md

**Comment by Nodragem** [comment]
Oooh, I did not see it I am sorry. As the first section of the README did not make sense to me I just skipped the rest.

The thing is as the beginning of the README did not make sense to me, I did not bother reading further.

Maybe reword the README into:

## How to Install
Follow this steps
1. download the source from Github and unzip
2. create the folders `libs/dragonBones/` in `Phaser/Demos`
3. copy-paste the content of `Phaser/2.x/out` in the folder `Phaser/Demos/libs/`

4. Now you should have the following project structure:
```
Your project
    |-- libs
        |-- dragonBones
            |-- dragonBones.js
            |-- ...
        |-- phaser
            |-- phaser.d.ts
            |-- pixi.d.ts
            |-- p2.d.ts
            |-- ...
    |-- node_modules
        |-- ...
    |-- out
        |-- ...
    |-- resource
        |-- ...
    |-- src
        |-- ...
    |-- ...
```
5. start a local server in `Phaser/Demos/` and go to visit `localhost:xxxx` in your favorite browser

**Comment by doubleclip** [comment]
Thanks Nodragem~
We will update the readme

**Comment by Nodragem** [comment]
Cool, :) thanks for the quick update!

Here are small improvements for steps 1 and 4, if you like it:

|1. download [DragonBonesJS](https://github.com/DragonBones/DragonBonesJS/archive/master.zip) from Github and unzip

|4. now you should have the following project structure:
```
DragonBonesJS/Phaser/Demos
    |-- libs
        |-- dragonBones
            |-- dragonBones.js
            |-- ...
        |-- phaser (optional, see note below*)
            |-- phaser.d.ts
            |-- pixi.d.ts
            |-- p2.d.ts
            |-- ...
    |-- node_modules (automatically generated when using npm)
        |-- ...
    |-- out
        |-- ...
    |-- resource
        |-- ...
    |-- src
        |-- ...
    |-- index.html
    |-- README.md
    |-- ...
```
**NOTE:** we are fetching Phaser with the package manager `npm` so you don't need to install it to run the demos. If you installed your own version of Phaser in `./libs/phaser/phaser.js`, you will need to modify the file `index.html`. Find the line `<script src="//cdn.jsdelivr.net/npm/phaser-ce@2.9.2/build/phaser.js"></script>`, and replace it to `<script src="./libs/phaser/phaser.js"></script>`

**Comment by doubleclip** [comment]
Thanks~ I have updated it.

**Comment by Nodragem** [comment]
cool :) I am happy that we could improve the doc together. 
Hope it will help future users!

I am closing the issue for now.
